### PR TITLE
replace [[wiki style links]] with markdown links

### PR DESCRIPTION
**Pull request by larsks**

### CODE REVIEW

File: docs/source/Install-Devstack-with-Sahara.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ cd devstack
 A `local.conf` file should be include into the devstack folder. 
 
 ```
-[[local|localrc]]
+[local](localr.html)
```
Comment (naved001) [inline docs/source/Install-Devstack-with-Sahara.md]: this shouldn't be replaced I'd imagine.

File: docs/source/Midonet-Kilo-Release.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ What devvesa meant was to click on "certain" patch such as https://review.openst
     `$ sudo vim local.conf`
 
          #!/usr/bin/env bash
-         [[local|localrc]]
+         [local](localr.html)
```
Comment (naved001) [inline docs/source/Midonet-Kilo-Release.md]: not a link

File: docs/source/archives-page/Install-Devstack-with-Vagrant.md [hunk 3]
```diff
@@ -67,7 +67,7 @@
 ### Configuration in Devstack
 Edit the devstack/localrc file to match the following (minimal configuration): 
 
-    [[local|localrc]]
+    [local](localr.html)
```
Comment (naved001) [inline docs/source/archives-page/Install-Devstack-with-Vagrant.md]: not a link

File: docs/source/clusters/kaizen/Midonet.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ Create a local.conf file and dump the below content to it.
 ```
 #!/usr/bin/env bash
 
-[[local|localrc]]
+[local](localr.html)
```
Comment (naved001) [inline docs/source/clusters/kaizen/Midonet.md]: not a link

**Comment by hgibson1** [comment]
I want to merge this one last after all the files have been moved.

**Comment by hgibson1** [comment]
Lars you can replace the links now, all files have been moved.

**Comment by larsks** [comment]
On Mon, Feb 04, 2019 at 06:26:39AM -0800, Hannah Gibson wrote:
> Lars you can merge this in now after fixing merge conflicts and false positives. All the files have been moved.

Sure, I'll try to get to that today or tomorrow.

-- 
Lars Kellogg-Stedman <lars@redhat.com> | larsks @ {irc,twitter,github}
http://blog.oddbit.com/                |

**Comment by hgibson1** [comment]
There were only 15 or so wiki links and I had nothing else to do so I went ahead and fixed them. You can close this.

**Comment by knikolla** [comment]
As Hannah said above, closing.

**Pull request closed without merging**